### PR TITLE
fix: add aws neccessary envs to circleci

### PR DESCRIPTION
**Pull request by KubaBaniak**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -46,6 +46,10 @@ commands:
             REDIS_PASSWORD="$REDIS_PASSWORD"
             REDIS_URL="$REDIS_URL"
             APP_IMAGE_NAME="$APP_IMAGE_NAME"
+            AWS_S3_BUCKET="$AWS_S3_BUCKET"
+            AWS_S3_ACCESS_KEY="$AWS_S3_ACCESS_KEY"
+            AWS_S3_KEY_SECRET="$AWS_S3_KEY_SECRET"
+            AWS_S3_REGION="$AWS_S3_REGION"
```
Comment (PrzerostNadSens) [inline .circleci/config.yml:52]:
```suggestion
            AWS_REGION="$AWS_REGION"
```

 in my, it is better to name it `AWS_REGION ` to be able to use this variable in a different context not just for S3

File: .circleci/config.yml [hunk 2]
```diff
@@ -46,6 +46,10 @@ commands:
             REDIS_PASSWORD="$REDIS_PASSWORD"
             REDIS_URL="$REDIS_URL"
             APP_IMAGE_NAME="$APP_IMAGE_NAME"
+            AWS_S3_BUCKET="$AWS_S3_BUCKET"
+            AWS_S3_ACCESS_KEY="$AWS_S3_ACCESS_KEY"
+            AWS_S3_KEY_SECRET="$AWS_S3_KEY_SECRET"
```
Comment (PrzerostNadSens) [inline .circleci/config.yml:51]:
```suggestion
            AWS_ACCESS_KEY_ID="$AWS_ACCESS_KEY_ID"
            AWS_SECRET_ACCESS_KEY ="$AWS_SECRET_ACCESS_KEY"
```
you should use names from the documentation
https://docs.aws.amazon.com/sdk-for-php/v3/developer-guide/guide_credentials_environment.html

**Review by PrzerostNadSens — APPROVED** [review]

File: .circleci/config.yml (hunk 2 above)
Comment (rad20c) [reply .circleci/config.yml:51]: I thought I already commented on this in a previous PR. Did you miss that comment?

Comment (KubaBaniak) [reply .circleci/config.yml:51]: Yep, I totally missed that by accident 😬

Comment (KubaBaniak) [reply .circleci/config.yml:51]: Ohh, I see. I only changed `example.env` file.

**Pull request merged**